City-of-Helsinki/drupal-hdbt
# UHF-X: Added missing comma to unit minimal display

# UHF-X: Added missing comma to unit minimal display
Added a missing comma to minimal unit display.

## What was done
* Added a missing comma to minimal unit display.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-X_fix_missing_comma`
* Run `make drush-cr`

## How to test
* Go to daycare search and make sure there is a comma between street address and the post number.

## Other PRs
*
